### ISSUE TITLE
WKT for collection doesn't do much

### ISSUE THREAD

**Issue by remster** [terranodo/tegola#227]
[Here](https://github.com/terranodo/tegola/blob/master/wkb/wkt.go#L108) is:
```
		for sg := range g.Geometries() {
			s := WKT(sg)
			geometries = append(geometries, s)
		}
```
should be:
```
		for _, sg := range g.Geometries() {
			s := WKT(sg)
			geometries = append(geometries, s)
		}
```
Essentially, I think you have a bit of a design problem. By the way you have defined your tegola.Geometry interface, everything fulfills it, even number seven. And your code (e.g.: here) is quite lenient producing empty wkt. Perhaps you want to add a token aspect of the geometry to the interface, like: PointCount() or Area() or Intersection(thought that would be more involved)

**Comment by gdey** [comment]
Thanks for this. This is a bug. The WKT was one of the first things I build and it was mainly to help debug things. I'm in the process of rewriting the WKB and WKT encoding libs. Yeah, the WKT(sg) function should have done a type check and returned an error. The issue is it returns an empty string when it does not recognize a type; instead of returning an error. https://github.com/terranodo/tegola/blob/master/wkb/wkt.go#L64
 I'll create a quite fix for this, now.

**Comment by gdey** [comment]
I'll am working through all wkb and wkt issues in the geom branch. Right now that has a better support for WKB in the [`tegola/geom/encoding/wkb`](https://github.com/terranodo/tegola/tree/geom/geom/encoding) directory. wkt support is still incomplete. Encode into WKT should work; I'm still working on decode as time allows.

**Comment by ARolek** [comment]
This fix is now merged into the v0.6.0 branch and will be part of the upcoming release. As @gdey mentioned, we're overhauling the wkb package which might be released with v0.6.0. We're also redefining the core geom types which the new wkb package will leverage. Thanks for the report!
